diam/autogen.sh

```
@@ -104,9 +102,10 @@ cat $dict | "$SED" \
 	-e 's/-Id\([-"s]\)/-ID\1/g' \
 	-e 's/-//g' \
 	-ne 's/.*avp name="\(.*\)" code="\([0-9]*\)".*/\1 = \2/p' \
+	| "$SED" -e 's/^5G/FiveG/' \
 	| LC_COLLATE=C sort -u $SORT_FLAG_IGNORE_CASE >> $src
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)